examples/REEL/README.md

```diff
@@ -0,0 +1,101 @@
+<img src="assets/reel_mock.png">
```

Comment:
I don't think we should add another folder `examples`, that would increase the complexity of the folder structure. One of the ideas that we were considering is to rename script folder to tools. Do you think tools would makes sense for this content?

Reply:
yeah, maybe a separate repo would make sense. We can discuss it tomorrow